modules/backend/assets/js/auth/uninstall-sw.js

```diff
@@ -1,7 +1,11 @@
-// Unregister all service workers before signing in to prevent cache issues
-navigator.serviceWorker.getRegistrations().then(
-    function(registrations) {
-        for (let registration of registrations) {  
-            registration.unregister();
-        }
-});
\ No newline at end of file
+// Wrapper to run the code only for HTTPs Connection
+if (location.protocol != 'https:')
+{
```

Comment:
curly brace on first line, also please capitalize the last `s` in `HTTPS` in the comment, lowercase `Connection` and pluralize it, comment should become `Only run on HTTPS connections`

Reply:
Done - Updated both as per your request.